client/components/school/school.service.js

```diff
@@ -2,8 +2,16 @@
 
 var app = angular.module('app');
 
-app.service('School', function($resource) {
-  return $resource('/api/schools/:id', {
+app.factory('School', function($resource) {
+  return $resource('/api/schools/:id/:controller', {
     id: '@_id'
+  }, {
+    students: {
+      method: 'GET',
+      isArray:true,
+      params: {
+        controller: 'students'
+      }
+    }
```

Comment:
https://docs.angularjs.org/api/ngResource/service/$resource

The second object is a custom call that will use `GET` with `:controller` replaced by `students` and still uses the default `id: '@_id'`, so the call will be made to `/api/schools/<id>/students`